stubs/model/model_import.stub

```diff
@@ -0,0 +1,4 @@
+// TODO: import relation models correctly
+// import { RelationModel } from '@lk-{{alias}}/src/models/RelationModel';
+
+import { useIncludesRepository } from '@lk-framework/src/repositories/IncludesRepository';
```

Comment:
swap them - include real file at the top, then TODO (so they will be last in imports)